vivo-project/VIVO
# Improve wording of intro-para1

**Scholars of All Disciplines**
* * *

**JIRA Issue https://jira.duraspace.org/browse/VIVO-1487**

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)

# What does this pull request do?
Changes the wording on the home page to

VIVO is a research-focused discovery tool that enables collaboration among scholars of all disciplines.

This changes removes the word "scientists" from the home page.  

# What's new?
Changes are made in the wilma and tenderfoot all.properties files, changing the text of intro_para1.

Changes are made in the EditSiteInfo.html page for acceptance testing.

# How should this be tested?
Build and view the home page.  You should see

VIVO is a research-focused discovery tool that enables collaboration among scholars of all disciplines.

# Additional Notes:
This change has been requested in various community discussions since 2012.  VIVO is about all scholarship, not just scientists.

Documentation showing the home page *may* include new screen shots to include the wording improvement.

# Interested parties
@VIVO-project/vivo-committers
